### PR TITLE
Add "alias" to conventions

### PR DESCRIPTION
**Pull request by mccv1r0**
Add "alias" to conventions

Signed-off-by: Michael Cambria <mcambria@redhat.com>

### CODE REVIEW

**Comment by mccv1r0** [comment]
Replace https://github.com/containernetworking/cni/pull/799

File: CONVENTIONS.md [hunk 1]
```diff
@@ -64,7 +64,8 @@ But the runtime would fill in the mappings so the plugin itself would receive so
...
 | infiniband guid | Dynamically assign Infiniband GUID to network interface. Runtime can pass this to plugins which need Infiniband GUID as input. | `infinibandGUID` | `GUID` (string entry). <pre> "c2:11:22:33:44:55:66:77" </pre> | none | CNI [`ib-sriov-cni`](https://github.com/Mellanox/ib-sriov-cni) plugin |
-| device id | Provide device identifier which is associated with the network to allow the CNI plugin to perform device dependent network configurations. | `deviceID` | `deviceID` (string entry). <pre> "0000:04:00.5" </pre> | none | CNI `host-local` plugin |
+| device id | Provide device identifier which is associated with the network to allow the CNI plugin to perform device dependent network configurations. | `deviceID` | `deviceID` (string entry). <pre> "0000:04:00.5" </pre> | none | CNI `host-device` plugin |
+| alias | Provide a list of names that will be mapped to the IP addresses assigned to this interface. Other containers on the same network may use one of these names to access the container.| `alias` | List of `alias` (string entry). <pre> my-container </pre> | none | CNI `alias` plugin |
```
Comment (squeed) [inline CONVENTIONS.md:68]: List of `alias` (string entries). <pre> [ "my-container", "primary-db"]  </pre>

**Comment by squeed** [comment]
LGTM

File: CONVENTIONS.md [hunk 2]
```diff
@@ -64,7 +64,8 @@ But the runtime would fill in the mappings so the plugin itself would receive so
...
 | infiniband guid | Dynamically assign Infiniband GUID to network interface. Runtime can pass this to plugins which need Infiniband GUID as input. | `infinibandGUID` | `GUID` (string entry). <pre> "c2:11:22:33:44:55:66:77" </pre> | none | CNI [`ib-sriov-cni`](https://github.com/Mellanox/ib-sriov-cni) plugin |
-| device id | Provide device identifier which is associated with the network to allow the CNI plugin to perform device dependent network configurations. | `deviceID` | `deviceID` (string entry). <pre> "0000:04:00.5" </pre> | none | CNI `host-local` plugin |
+| device id | Provide device identifier which is associated with the network to allow the CNI plugin to perform device dependent network configurations. | `deviceID` | `deviceID` (string entry). <pre> "0000:04:00.5" </pre> | none | CNI `host-device` plugin |
+| alias | Provide a list of names that will be mapped to the IP addresses assigned to this interface. Other containers on the same network may use one of these names to access the container.| `alias` | List of `alias` (string entry). <pre> ["my-container", "primary-db"] </pre> | none | CNI `alias` plugin |
```
Comment (mars1024) [inline CONVENTIONS.md:68]: Since this field is a list, should we name this "aliases", just like "ips"?

Comment (dcbw) [reply CONVENTIONS.md:68]: @mccv1r0 @mars1024 I would agree, we should do 'aliases' I think

**Pull request merged**